site/docs/windows.md

```diff
@@ -90,7 +92,9 @@ To build C++ targets, you need:
     The first command sets the path to Visual Studio (BAZEL\_V<b>S</b>), the other
     sets the path to Visual C++ (BAZEL\_V<b>C</b>).
 
-    For Visual Studio 2017, with a default install, instead you might want
+    For Visual Studio 2017, you might have ambiguity with VC++ version.
```

Comment:
"For Visual Studio 2017, you must specify the target VC++ version via `BAZEL_VC_TOOL` if more than one VC++ version is installed."